hack/patch-nix-hash.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
```

Comment:
we should have a comment at the top of this script describing what it does / why it exists / needs to exist